lang/java/reef-wake/wake/src/test/java/org/apache/reef/wake/test/time/ClockTest.java

```diff
@@ -33,21 +35,28 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.Random;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 import java.util.logging.Level;
 
 /**
- * Tests for Clock.
+ * Tests for RuntimeClock event loop.
  */
 public class ClockTest {
```

Comment:
If this is the test for the RuntimeClock event loop, shouldn't the test class name reflect it ?